developer/deploying/platform.md

```diff
@@ -0,0 +1,179 @@
+
+### register
+
+`reaction register` and create keys locally before configuring your application environment. `reaction login` authenticates you to the Reaction Platform and syncronizes platform services.
+
+```sh
+# Register with invite token
+reaction register
+
+# or if you've already registered, login with your username and password
+reaction login
+```
+
+### keys
+
+You will need to set up an SSH key to securely communicate with Launchdock.
+
+Set up an SSH key pair:
+
+```sh
+# https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
+#
+# create a new SSH key pair
+# prompts for filename
+# suggest "~/.ssh/launchdock" for ease
+ssh-keygen -t rsa -b 4096 -C "you@example.com"
+
+# make sure the ssh-agent is running in the background
+eval "$(ssh-agent -s)"
+
+# add your new key to the agent
+ssh-add -K ~/.ssh/<private key created above>
+
+# add your public key to Launchdock
+reaction keys add ~/.ssh/<keyname>.pub
+```
+
+## Applications
+
+An "application" is a instance of Reaction Commerce and any services, connectors, or container dependencies that are configured to deploy as one to the application environment.
+
+### create
+
+Creating an "application" configures the environment that the built application will be deployed into.
+
+```sh
+# Create and configure your new app deployment
+reaction apps create \
+  --name <appname> \
+  -e REACTION_USER="yourname" \
+  -e REACTION_AUTH="P@s5w0rd" \
+  -e REACTION_EMAIL="you@example.com" \
+  --registry  path/to/reaction.json \
+  --settings path/to/settings.json
+```
+
+#### env
+
+Configure environment variables for the application deployment.
+
+```sh
+# set/update environment variables
+# (this triggers a redeploy of your app)
+reaction env set \
+  --app <appname> \
+  -e SOME_API_KEY="ec89jmur3jim8e34" \
+  -e SOME_OTHER_THING="dj8dr34ju3r@#$" \
+  -e MAIL_URL="smtp://USERNAME:PASSWORD@NEW_HOST:PORT"
+
+# unset environment variables
+# (this triggers a redeploy of your app)
+reaction env unset --app <appname> -e SOME_API_KEY -e SOME_OTHER_THING
+
+# list your currently set environment variables
+reaction env list --app <appname>
+```
+
+**--registry** pass a `reaction.json` to the environment
+
+**--settings** pass a Meteor `settings.json` to the environment
+
+#### domains
+
+```sh
+# add a custom domain for your app
+# (first, update your DNS to point your domain at your app's default URL)
+reaction domains add --app <appname> -d mycoolshop.com
+
+# remove a custom domain from your app
+reaction domains remove --app <appname> -d mycoolshop.com
+
+#### apps list
+# list your apps and their domains
+reaction apps list
+```
+
+### deploy
+
+Submit your application for a deployment. This will begin the CI/CD/CD process. This begins a build process that is unique for every project. Expect this entire process to take around 20 minutes.
```

Comment:
spell out abbreviations on first reference for `CI/CD/CD`

Reply:
I'd like to make it more clear that we're also referring to "integration, delivery, and deployment", as opposed to "integration and deployment".